vllm/model_executor/layers/layernorm.py

```diff
@@ -95,3 +95,49 @@ def extra_repr(self) -> str:
         s = f"hidden_size={self.weight.data.size(0)}"
         s += f", eps={self.variance_epsilon}"
         return s
+
+
+class GemmaRMSNorm(CustomOp):
+    """RMS normalization for Gemma.
```

Comment:
Should this be named `Gemma2RMSNorm` for clarity? It looks like `gemma.py` uses the regular rms norm

Reply:
Actually Gemma1 also has to use `GemmaRMSNorm`. This will be fixed in #5913